captain/routes/flowchart.py

```diff
@@ -65,7 +65,7 @@ async def write_and_run_flowchart(request: PostWFC):
 async def worker_response(
     request: Request,
 ):  # TODO figure out way to use Pydantic model, for now use type Request otherwise does not work????
-    if manager.running_topology is None or manager.running_topology.is_cancelled():
+    if manager.running_topology is None or manager.running_topology.cancelled:
```

Comment:
Seems annoying, but better to have a method for this, we want to follow the paradigm where we treat topology like a separate object/model and not something we can access its attributes easily.

Reply:
Yes, makes sense. bringing that method back.